cli/src/main.rs

```diff
@@ -99,6 +99,7 @@ pub fn parse_args(matches: &ArgMatches<'_>) -> Result<WalletConfig, Box<dyn erro
         command,
         json_rpc_url,
         keypair,
+        keypair_location: id_path.to_string(),
```

Comment:
nit: keypair_path feels a little more descriptive (and shorter ftw) than keypair_location

Reply:
Oh duh, that's def better.